invenio_banners/records/models.py

```diff
@@ -62,22 +63,21 @@ def create(cls, data):
             )
             db.session.add(obj)
 
-        db.session.commit()
```

Comment:
We need to add the `unit of work` decorator in each service method that is modifying a record, and therefore calling these create/update/delete.

Reply:
this db.session.commit is not necessary. it is also not necessary to change something in the services.

tried in a instance. the banner is stored in the database also without that db.session.commit()

Reply:
It cannot be magically happen... a decorator must be added [here](https://github.com/inveniosoftware/invenio-banners/blob/master/invenio_banners/services/service.py#L84)
If it happens, we need to understand why.
Maybe [flask-sqlalchemy](https://github.com/pallets-eco/flask-sqlalchemy) is auto-committing and the end of the HTTP request?

Reply:
To add a bit to the confusion: [Here](https://github.com/pallets-eco/flask-sqlalchemy/blob/3e3e92ba557649ab5251eda860a67656cc8c10af/CHANGES.rst#L201-L202) it seems that they move towards explicit `db.session.commit`'s.

Reply:
@ntarocco you are totally right, it cannot magically happen. it happened the first time i tested it but i couldn't reproduce it. so i added now the unit_of_work decorator to all service methods which create/update/delete models

Reply:
Nice thanks! In case you want to be 100% sure, you could change the `resources` tests and add something like this:
```python
id = client.put(...)  # create banner
client.get(.../<id>)  # get the created banner
```

This should fail without the `uow`, and pass after adding it ☺️